WNPRC_EHR/src/org/labkey/wnprc_ehr/table/WNPRC_EHRCustomizer.java

```diff
@@ -1476,22 +1476,30 @@ public DisplayColumn createRenderer(ColumnInfo colInfo)
             String num_animals_assigned = "num_animals_assigned";
             TableInfo assignmentTable = getRealTableForDataset(table, "assignment");
 
-            String theQuery  = "( " +
-                    "(SELECT " +
-                    "(CASE WHEN COUNT(*) = 0 " +
-                    " THEN null " +
-                    " ELSE COUNT(*) " +
-                    " END) as num_animals_assigned " +
-                    " FROM studydataset." + assignmentTable.getName() + " a " +
-                    "WHERE a.animal_request_rowid=" + ExprColumn.STR_TABLE_ALIAS + ".rowid )  " +
-                    ")";
-
-            SQLFragment sql = new SQLFragment(theQuery);
+            // Adding null check here to keep teamcity happy otherwise 'WNPRCComplianceTrainingTest.testSopSubmission' test
```

Comment:
I'd add to this comment to mention this test doesn't bother to set up a full EHR folder, which is why it can't find the assignment table.

Reply:
Good point, done!